41.md

```diff
@@ -0,0 +1,69 @@
+NIP-XX
+======
+
+Surveys / polls
+-----------------------------------
+
+`draft` `optional` `require:nip-20` `author:fernandolguevara`
+
+The `surv` & `surv-resp` tags enables users to have surveys/polls experiences on the network.
+
+
+#### Spec
+
+##### Message
+
+```
+tag: surv
+options:
+ - <multi|single> allow others to reply with one or multiple options
+ - <ttl> TTL (in seconds|timestamp) when surv expires, 0 TTL don't expire
+ - [<choice>]: up to 4 choices each limited to 25 chars  
+````
+
+```
+tag: surv-resp
+options:
+ - [<choice>]: based on surv type it can have one or multi
```

Comment:
{
    "pubkey": "<pub-key>",
    "created_at": 1000000000,
    "kind": 8,
    "tags": [
      ["e", "<surv-event-id>", "wss://..."], // Should be the only e event which is the surv event
      ["surv-resp", "1", "2"...] // based on root event surv type it can have one or multi 
    ],
    "content": "hello #tag\n",
    "id": "<event-id>"
  }

```suggestion
tag: surv-resp
kind: 8
options:
-  [<e tag poll/survey event id>]: Should be the only e event which is the surv event
 - [<choice>]: based on surv type it can have one or multi, They represent the indexes of the answers
```